### ISSUE TITLE
BusinessChartBuffer.Update cannot convert CurrPage.Chart 

### ISSUE THREAD

**Issue by djole1391** [Microsoft/AL#2477]
Hello,

Now that I installed container with latest image and new AL extension from .vsix, I have an error for chart that wasn't there earlier:

> Argument 1: cannot convert from 'UserControl Chart: "Microsoft.Dynamics.Nav.Client.BusinessChart"' to '__MissingTypeSymbol__'

This is where this functions is being called:

```
local procedure UpdateChart();
    begin
        ActualBudgetChartMgt.GenerateData(BusinessChartBuffer);
        BusinessChartBuffer.Update(CurrPage.Chart);
    end;
```
And this is how Chart usercontrol is declared:

`usercontrol(Chart;"Microsoft.Dynamics.Nav.Client.BusinessChart")
`
AL extension versions is 0.17.19506. 

Any input is welcome.

Thanks!

**Comment by atoader** [comment]
Hi! I am guessing you are using the latest docker drop. The quick solution is to set the following setting:
'al.assemblyProbingPaths':[{PathToAddInsFolder}]

where PathToAddInsFolder is the path to your server's Add-ins folder. I am working on docs for this feature.

**Comment by djole1391** [comment]
I can't see server add ins folder, only client. 

I tried mapping this folder to my folder using additional parameter during creation of new container by adding following parameter to new-navcontainer command :

> 
-additionalParameters @('-v "c:\myaddins:c:\program files\microsoft Dynamics NAV\110\Service\Add-ins"')

but I kept getting some errors. Maybe my syntax is wrong? Or if you have some other idea on how I can get access to server addins folder.

Thanks!
